### PR TITLE
Support Z690 Aorus Pro (DDR5)

### PR DESCRIPTION
**Pull request by zokradonh**
- Improved IT8689E 
- Added IT8695E
- Added Mainboard Gigabyte Z690 Aorus Pro (DDR5)

Introduced Multi Bank support of chip IT8689E.
To control fans of IT8689E we may need to handle the multi banks.

- [x] Support control of fans of IT8689E
- [ ] Support control of fans of IT8695E

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Chip.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ internal enum Chip : ushort
         IT8686E = 0x8686,
         IT8688E = 0x8688,
         IT8689E = 0x8689,
+        IT8695E = 0x8695, // name unclear. May be also IT8795E
```
[the remark this answers is not in the document]
Comment (zokradonh) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Chip.cs:38]: yes, ok. I will remove the comment from source code.

**Comment by sebastian-dev** [comment]
#define IT8792E_DEVID 0x8733
https://github.com/torvalds/linux/blob/master/drivers/hwmon/it87.c

**Comment by zokradonh** [comment]
You mean, maybe 0x8695 == IT8795E ?
because 0x8733 can not be IT8795E because 0x8733 == IT8792E according to linux kernel?

**Comment by sebastian-dev** [comment]
> ou mean, maybe 0x8695 == IT8795E ?
> because 0x8733 can not be IT8795E because 0x8733 == IT8792E according to linux kernel?
my mistake, i have compared some wrong numbers

**Comment by zokradonh** [comment]
I think correct fan control of IT8695E requires physical RAM read/write via Gigabyte GIO driver. I may succeed here sooner or later but I think it may be better to pull the support of IT8689E in the meantime since other mainboards also use this chip.

**Comment by PhyxionNL** [comment]
> I think correct fan control of IT8695E requires physical RAM read/write via Gigabyte GIO driver. 
That seems unlikely, but we can remove this if it's not working correctly.

**Comment by zokradonh** [comment]
I have changed code according to your requested changes.

**Comment by PhyxionNL** [comment]
Thanks for this PR!

**Pull request merged**